test/hotspot/jtreg/runtime/SharedArchiveFile/MaxMetaspaceSize.java

```diff
@@ -45,12 +47,11 @@ public static void main(String[] args) throws Exception {
     if (Platform.is64bit()) {
       processArgs.add("-XX:MaxMetaspaceSize=3m");
       processArgs.add("-XX:CompressedClassSpaceSize=1m");
-      processArgs.add("-XX:InitialBootClassLoaderMetaspaceSize=1m");
     } else {
       processArgs.add("-XX:MaxMetaspaceSize=1m");
     }
 
-    String msg = "Failed allocating metaspace object";
+    String msg = "OutOfMemoryError: Metaspace";
     ProcessBuilder pb = ProcessTools.createJavaProcessBuilder(processArgs);
```

Comment:
Includes the changes from [8261551](https://bugs.openjdk.org/browse/JDK-8261551) (commit [41657b1](https://github.com/openjdk/jdk/commit/41657b15ac8322cc109d7f6105873ae388e48ff3) ) on this file